lambdatimes/src/components/Content/Tabs.js

```diff
@@ -1,16 +1,25 @@
 import React from 'react';
 import Tab from './Tab';
+import propTypes from 'prop-types';
+
 const Tabs = props => {
   return (
     <div className="tabs">
       <div className="topics">
         <span className="title">TRENDING TOPICS:</span>
+         {props.tabs.map( (tab, index) =>
+          <Tab tab = {tab} selectedTab = {props.selectedTab} selectTabHandler = {props.selectedHandler} key = {index} />
```

Comment:
general reminder not to use keys for indexes, though I know you basically had to here